mods/d2k/rules/atreides.yaml

```diff
@@ -1,282 +0,0 @@
-CONYARDA:
-	Inherits: ^CONYARD
-
-PWRA:
-	Inherits: ^POWER
-	Buildable:
-		Prerequisites: ~conyarda
-
-WALLA:
-	Inherits: ^WALL
-	Buildable:
-		Prerequisites: ~conyarda, barracks
-
-GUNTOWERA:
-	Inherits: ^GUNTOWER
-	Buildable:
-		Prerequisites: ~conyarda, barracks
-
-ROCKETTOWERA:
-	Inherits: ^ROCKETTOWER
-	Buildable:
-		Prerequisites: ~conyarda, radar
-
-REFA:
-	Inherits: ^REFINERY
-	Buildable:
-		Prerequisites: ~conyarda, power
-
-BARRA:
```

Comment:
Actually on bleed `^BARRACKS:` is the one with 2 exits, so all faction's barracks have 2. As will they now.